src/main/java/use_cases/session_load/SessionLoadOD.java

```diff
@@ -0,0 +1,31 @@
+package use_cases.session_load;
+
+import entities.SessionStorage;
+
+public class SessionLoadOD {
+    private final String message;
+    private final SessionStorage session;
+
+    public SessionLoadOD(String message, SessionStorage session) {
+        this.message = message;
+        this.session = session;
+    }
+
+    /**
+     * Gets the message to display
+     *
+     * @return a String containing the message to display
+     */
+    public String getMessage() {
+        return message;
+    }
+
+    /**
+     * Gets the session data that was loaded
+     *
+     * @return a SessionStorage object with the loaded data
+     */
+    public SessionStorage getSession() {
+        return session;
+    }
+}
```

Comment:
Follow Intellij suggestions to add setters. Although we might not need it, it is just a convention to have getters and setters that follows the Open/Closed Principle.

Reply:
Are you sure? I don't really see the need in adding methods that we aren't gonna use, it would just flag warnings in Intellij. Was this something the prof or TA said?

Reply:
This is just a convention that code writers do in real life. This is similar to the "Write List<Expenses> = new ArrayList... instead of ArrayList<Expenses> = new ArrayList..." that I mentioned in the group chat. Its not wrong as the way it is, but using List is more professional as it follows Open/Closed Principle. Similarly, it's not wrong to not have the setters, just that conventionally, you also create the setters so that any people who want to use our code can access the Entities without changing our code.

Reply:
Well in this case the output data should just be modified by the presenter when creating it no? So adding public setters would just add the ability to modify the already created output data when it shouldn't actually be modified after its creation, so I kinda disagree on adding the setters. I'll wait for everyone to comment here so that we can decide and I can add them pretty quick if we need them.

Reply:
I agree with you, Ari. It's just two different views where some think the code should be open to extension (especially so in the case of making a library); some think the code should be clean and minimal (especially when the same person will continue update the program). 

In our case, we are not making a library, and it is likely that only we will make changes to the code, so we do not need to write the setters. However, it will be our group design decision to include setters or not, so we must reach a consensus and make sure all use cases does the same.

Reply:
As I mentioned in our gc, I will need setters for my input data to cast the user input of category budgets and expense amounts. (since we allow users to input any number type but we are handling them in terms of doubles.)

Reply:
Conclusion made from group chat: Only add setters in input/output data classes if they are required for the feature. 

I have removed the setters from my input/output data classes too.